scripts/install-windows-prereqs.ps1

```diff
@@ -28,7 +29,8 @@ Param(
     [string]$AzureDCAPNupkgHash = '152ACE956348E80E533E63E6CB1D3CA20E4CA7DC775FC0B9413F552368F971D6',
     [Parameter(mandatory=$true)][string]$InstallPath,
     [Parameter(mandatory=$true)][bool]$WithFLC,
-    [Parameter(mandatory=$true)][bool]$WithAzureDCAPClient
+    [Parameter(mandatory=$true)][bool]$WithAzureDCAPClient,
+    [switch]$WithoutDriver
```

Comment:
Maybe, from a consistency perspective, I should make this a bool, like the others, rename it to "WithDriver", and default it to $true (without a mandatory flag).

Reply:
Actually, I believe the standard preference is to use switches and structure bool defaults to `$false`.

Given the growing number of options, particularly given that some of them have interactions (i.e. Installing Azure DCAP client without FLC makes no sense), you might want to structure this as a single parameter taking an enum (or an array of enum values if you want to allow arbitrary combinations) instead.

Reply:
Good idea, thanks Simon.